proto/s2s.go

```diff
@@ -65,9 +67,11 @@ type RequestArg struct {
 // JobLog represents a log entry for a finished job.
 type JobLog struct {
 	// These three fields uniquely identify an entry in the job log.
-	RequestId string `json:"requestId"`
-	JobId     string `json:"jobId"`
-	Try       uint   `json:"try"` // try number N of 1 + Job.Retry
+	RequestId   string `json:"requestId"`
+	JobId       string `json:"jobId"`
+	Try         uint   `json:"try"`         // try number N of 1 + Job.Retry + SequenceTry
```

Comment:
Same issue here wrt job try being monotonic

Reply:
`s/ + SequenceTry//`

Reply:
I replaced it with `try number that is monotonically increasing` because `1 + Job.Retry` is no longer true (can be greater than 1+ Job.Retry if the job maxes out retries then it tried again as part of a  sequence retry).